resources/puppetlabs/lein-ezbake/template/global/Gemfile

```diff
@@ -0,0 +1,13 @@
+source ENV['GEM_SOURCE'] || 'https://artifactory.delivery.puppetlabs.net/artifactory/api/gems/rubygems/'
+
+def location_for(place, fake_version = nil)
+  if place =~ /^(git[:@][^#]*)#(.*)/
+    [fake_version, { :git => $1, :branch => $2, :require => false }].compact
+  elsif place =~ /^file:\/\/(.*)/
+    ['>= 0', { :path => File.expand_path($1), :require => false }]
+  else
+    [place, { :require => false }]
+  end
+end
+
+gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '~> 0.99')
```

Comment:
will this pull in the right version? I know there were some bundler issues that caused us to use a more specific version binding for some projects

Reply:
yeah, when I bundle install with the ^build I did, it pulls in packaging 0.99.10.
since this will be used by multiple projects, I was hoping to make this as broad as possible, to minimize updates.